Feature/#206/redefine small design changes

Ccolina03 opened the pull request:
-Home Page only allows 3 vendors now

Figma Ans:
-Font size of 'Welcome to ISA's mobile app' is 35 px
-Spacing not 16px. 
-Icon moved to 6.5
-Font size for Vendor Cards info changed to 16,14 and 12 for title, address, and description, respectively (Added +2 to each to match better my screen to the figma)
-Add spacing between address and description
-Added shadow in line for tab bar
-Font size in tab bar icon names to 12
-Adding spacing to Vendors in top (Search Bar page)
-Matching search bar horizontal marginal to 24, as well as Vendors
-Search bar size, position, border radius to 15 (figma)
-Notification text in My ICard is 12 px
-Changed color red to match figma color red
-Font Size of userName and ISAFStatus (Inactive) in Icard is 26 px


Images: 

![image](https://user-images.githubusercontent.com/107608945/223659916-a78e7711-b532-4648-9e69-b6d18b85601b.png)

![image](https://user-images.githubusercontent.com/107608945/223659974-638ffc0a-d0ef-4874-a467-3b2538011ae3.png)

![image](https://user-images.githubusercontent.com/107608945/223660066-873e66fa-e2d3-49ac-b43e-f8abc02968bb.png)

![image](https://user-images.githubusercontent.com/107608945/223660125-f92819d7-a2df-4801-a920-63fe55c0aca3.png)

![image](https://user-images.githubusercontent.com/107608945/223660186-13b009e2-b171-43a1-8db7-835458a62c90.png)

Ccolina03 committed:
Limit of 3 vendors in HomeP and defenite of design features

Ccolina03 committed:
Limiting vendors in HomeP and redefining design features

Ccolina03 committed:
Updating Unlinked status back

ArmaanKatyal commented:
Can you please remove the changes to `backend/yarn.lock` that might break the backend. As this PR has no relation to the backend. Thanks

ArmaanKatyal committed:
Delete package-lock.json

why is package-lock.json required in the main directory? I think this file is added here by mistake

ArmaanKatyal commented:
I have deleted `package-lock.json` that was being added to the main directory for no reason. If required can be pushed back again.

ArmaanKatyal commented on mobile/src/views/HomeView.jsx:
```diff
@@ -56,9 +57,10 @@ const HomeView = ({navigation}) => {
         <VendorList
           searchPhrase={searchPhrase}
-          data={vendorData}
+          data={vendorData.slice(0,3)}
```
why do we need to slice the vendor data?

Ccolina03 replied:
To limit to 3 the amount of Vendor Cards appearing in HomePage

ArmaanKatyal commented on mobile/src/components/shared/VendorList.jsx:
```diff
@@ -49,22 +49,28 @@ const VendorList = ({searchPhrase, setClicked, data}) => {
+      },
+    });
 
 export default VendorList;
```
```suggestion
export default VendorList;

```

ArmaanKatyal commented on mobile/src/components/shared/SearchBar.jsx:
```diff
@@ -59,32 +59,34 @@ const SearchBar = ({clicked, setClicked, searchPhrase, setSearchPhrase}) => {
-        width: "90%",
+        width: "87%",
+        height: 40,
+        
```
nit: blank lines can de deleted. Use Prettier to format the code(preferred)

ArmaanKatyal commented on mobile/src/views/HomeView.jsx:
```diff
@@ -10,7 +10,7 @@ import {options, url} from '../data/vendorMockData';
 import {getData} from '../data/vendorMockData';
 import {SafeAreaView} from 'react-native-safe-area-context';
-
+import {ScrollView} from 'react-native-virtualized-view'
```
nit: Formatting
```suggestion
import {ScrollView} from 'react-native-virtualized-view'

```

ArmaanKatyal commented:
One more thing, only try to push the files that you have changed in the PR. Try not to push package-lock or yarn.lock because then it can cause breaking changes impossible to find as well as the lines changes is in 10,000 😂

Ccolina03 committed:
Delete yarn.lock

Ccolina03 commented:
> Can you please remove the changes to `backend/yarn.lock` that might break the backend. As this PR has no relation to the backend. Thanks



> One more thing, only try to push the files that you have changed in the PR. Try not to push package-lock or yarn.lock because then it can cause breaking changes impossible to find as well as the lines changes is in 10,000 😂

Yes, my bad I was playing around with the backend and forgot that I did that long ago. Should I pr again with the changes?

ArmaanKatyal commented:
> 

You don't have to create a new PR again. just remove the changes that are not required. 👍🏽

Ccolina03 committed:
Changes in response to PR

Ccolina03 committed:
Update VendorList.jsx

Ccolina03 committed:
Delete yarn.lock

ArmaanKatyal reviewed (approved):
lgtm

Ccolina03 committed:
Adding yarn.lock from mobile again

yismailuofa commented:
@ArmaanKatyal So to be clear we want to keep the `mobile/yarn.lock` but delete the `backend/yarn.lock` ?

ArmaanKatyal commented:
> @ArmaanKatyal So to be clear we want to keep the `mobile/yarn.lock` but delete the `backend/yarn.lock` ?

Yes, carlos by mistake pushed his experimental changes for backend when he was playing around with it. 
His PR is related to frontend so their shouldn't be any changes related to backend. 
@yismailuofa

yismailuofa commented:
> Yes, carlos by mistake pushed his experimental changes for backend when he was playing around with it.

So we need to remove backend/yarn.lock from the changes then right?

ArmaanKatyal commented:
> > Yes, carlos by mistake pushed his experimental changes for backend when he was playing around with it.
> 
> 
> 
> So we need to remove backend/yarn.lock from the changes then right?

Yes 👍

yismailuofa commented on backend/yarn.lock:
As per @ArmaanKatyal we shouldn't delete this, so let's take it out of this PR. Otherwise lgtm

Ccolina03 committed:
Adding unmodified yarn.lock
